### DISCUSSION TITLE
Any way to remove tiles on web page?

### DISCUSSION THREAD

**Discussion by Bugman1400** [aqualinkd/AqualinkD#254] [Q&A]
I don't have a spa so I was looking for ways to remove all spa related buttons on the web page. This includes when it flashes up on the top banner (ie Spa pump is Off).

**Comment by sjames99** [comment]
Check out the Web UI Config section of the wiki.   You can simple comment out the devices you don't have or want to appear.

https://github.com/sfeakes/AqualinkD/wiki#web-config

**Comment by Bugman1400** [comment]
That seemed to work for most of the buttons, but I still have the Spa Water Temp and the pesky banner indicating the Spa pump is off.

![image](https://github.com/sfeakes/AqualinkD/assets/25798328/c42eeb2e-900e-4fc3-9579-5136c16935a1)
